tensorboard/components/tf_dashboard_common/data-loader-behavior.ts

```diff
@@ -146,40 +142,39 @@ namespace tf_dashboard_common {
       }
     },
 
-    _loadDataImpl() {
+    async _loadDataImpl() {
       if (!this.active) return;
-      this.cancelAsync(this._loadDataAsync);
-      this._loadDataAsync = this.async(() => {
```

Comment:
you may want to keep this async around as it ties better with Polymer lifecycle.